app/server/utils/misc_tasks.py

```diff
@@ -3,6 +3,7 @@
 class MiscTasker(object):
     @staticmethod
     def set_ip_location(ip_address_id, ip):
+        # TODO: Standardize this task (pipe through execute_synchronous_celery)
```

Comment:
Just limiting scope of this PR to blockchain_tasks for now. Would like to follow this up with another PR for other worker tasks if that's fine